recipes-graphics/vulkan/vulkan-loader_%.bbappend

```diff
@@ -1,3 +1,3 @@
 # The i.MX implementation is dynamically loaded, so it requires an
 # explicit runtime dependency.
-RRECOMMENDS:${PN}:append:imxgpu = " libvulkan-imx"
+RRECOMMENDS:${PN}:imxvulkan += "libvulkan-imx"
```

Comment:
Don't mix override and `+=`. Use `:append` please.

Reply:
Sorry! fixed

Reply:
The main recipe has this:

`RRECOMMENDS:${PN} = "mesa-vulkan-drivers"`

We've been doing our releases like this, but I wonder if this was an oversight and should really be replaced by using assignment here instead of append:

`RRECOMMENDS:${PN}:imxvulkan = "libvulkan-imx"`

I can ask Prabhu.